atlasdb-cassandra-integration-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServiceIntegrationTest.java

```diff
@@ -155,7 +117,7 @@ protected boolean reverseRangesSupported() {
     @Override
     @Ignore
     public void testGetAllTableNames() {
-        //
+        // this test class creates a number of tables
```

Comment:
what on earth, can we delete this?

Reply:
The abstract class test depends on the fact that there is no pollution from other tests. The invariant does not hold for this implementation, so the test is ignored.

Could get rid of the test altogether or make sure the invariant holds, but had to stop refactoring somewhere